src/ignitus-Shared/ignitus-DesignSystem/ignitus-Atoms/colors.ts

```diff
@@ -20,11 +20,14 @@ export const Green: string = '#209653';
 export const Yellow: string = '#ffd747';
 
 // social platform colors
-export const TwitterIconColor: string = ' #00acee';
+export const TwitterIconColor: string = '#00acee';
 export const LinkedinIconColor: string = '#0e76a8';
-export const RedditIconColor: string = ' #FF4301';
+export const RedditIconColor: string = '#FF4301';
 export const AngellistIconColor: string = '#040404';
-export const FacebookIconColor: string = ' #3b5998';
+export const FacebookIconColor: string = '#3b5998';
+
+// other colors
+export const Orange: string = '#ffa500';
```

Comment:
Could you please use secondary color instead of this orange and remove this.

Reply:
But orange is a secondary colour, I guess. If not, then which colour should I replace it with?

Reply:
Yes, orange is the secondary color